concepts/configuration/README.md

```diff
@@ -1,8 +1,140 @@
-# Configuration
+# Configurations
+Dapr configurations are settings that enable you to change the behavior of individual Dapr sidecars or globally on the system services in the Dapr control plane.
 
-Dapr configuration is a configuration file (in local mode) or a Kubernetes configuration object (in Kubernetes mode). A Dapr sidecar can apply a configuration by using a ```dapr.io/config``` annotation (in Kubernetes mode) or by using a ```--config``` switch with ```dapr run```.
+An example of a per Dapr sidecar setting is configuring trace settings. An example of a control plane setting is mutual TLS which is a global setting on the Sentry system service.   
 
-A Dapr configuration configures:
+- [Self hosted sidecar configuration](#Self-hosted-sidecar-configuration)
+- [Kubernetes sidecar configuration](#Kubernetes-sidecar-configuration)
+- [Sidecar Configuration settings](#sidecar-configuration-settings)
+- [Kubernetes control plane configuration](#Kubernetes-control-plane-configuration)
+- [Control plane configuration settings](#control-plane-configuration-settings)
 
-* [distributed tracing](../observability/traces.md)
-* [custom pipeline](../middleware/README.md)
+## Self hosted sidecar configuration  
+In self hosted mode the Dapr configuration is a configuration file, for example `myappconfig.yaml`. By default Dapr side looks in the `components/` sub-folder under the folder where you run your application for a configuration file.
+

```

Comment:
Would be good to also mention that when running via the CLI, it will create the components directory with default configurations if the directory is not found